pymysql/tests/test_issues.py

```diff
@@ -381,3 +383,30 @@ def test_issue_175(self):
                     warnings.filterwarnings("ignore")
                     cur.execute('drop table if exists test_field_count')
 
+    def test_pull_request_for_fix_tuple_and_unicode_encoding_issue(self):
+        """ Encoding error appears when iterable is passed in query args list.
+        List should contain two types of items:
+        iterable containing unicode string and unicode string with non-ascii symbol"""
+        table_name = 'tuple_and_unicode'
+        connection = pymysql.connect(charset="utf8", **self.databases[0])
+        cursor = connection.cursor()
+        try:
+            cursor.execute('create table %s (value_1 varchar(1), value_2 varchar(1))' % table_name)
```

Comment:
I did not found `self.create_table()` in the test_issue_33
Did you mean DatabaseTest.create_table(). So could I inherit TestGitHubIssues from this class or move my test into it?